python/ribasim/ribasim/model.py

```diff
@@ -209,7 +199,7 @@ def _apply_crs_function(self, function_name: str, crs: str) -> None:
         self.crs = crs
 
     def node_table(self) -> NodeTable:
-        """Compute the full NodeTable from all node types."""
+        """Compute the full sorted NodeTable from all node types."""
```

Comment:
Sorted on what? And does it matter for this function?

Reply:
`sort_keys = ["node_type", "node_id"]`. I thought that the sort order was documented for this table, but it isn't, so I'll add it to my doc PR.

It matters because if it is documented to be sorted we know we don't have to sort it ourselves after. I now rely on this behavior in the code.